samples/WebAPI/WebAPI.csproj

```diff
@@ -9,8 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DurableTask.Client" Version="0.4.1-beta" />
-    <PackageReference Include="Microsoft.DurableTask.Generators" Version="0.4.1-beta" />
+    <ProjectReference Include="$(SrcRoot)Worker/Grpc/Worker.Grpc.csproj" />
+    <ProjectReference Include="$(SrcRoot)Client/Grpc/Client.Grpc.csproj" />
+    <ProjectReference Include="$(SrcRoot)Generators/Generators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
Previously the generated was bundled with the SDK nuget package. Any particular reason to separate it out in your refactor?

Reply:
It is bundled with it, but that bundling only works property via `PackageReference`. P2p reference needs to be explicit.